packages/ui-core/src/components/Notification/Notification.test.tsx

```diff
@@ -1,26 +1,51 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
-import Notification, { NotificationTypes } from './Notification';
+import Notification, { INotificationProps, NotificationTypes, ShadowTypes } from './Notification';
+import Attention from 'icons/System/16/Attention_16.svg';
+
+const props: INotificationProps = {
+    className: 'notification',
+    type: NotificationTypes.ERROR,
+    shadowLevel: ShadowTypes.HOVER,
+    href: 'href',
+    hasCloseButton: false,
+    rel: 'noopener',
+    target: '_blank',
+    icon: <Attention />,
+    title: 'title',
+    isColored: true,
+    content: 'content',
+};
 
 describe('<Notification />', () => {
-    it('it renders Notification without props', () => {
+    it('renders component', () => {
+        const icon = <Attention />;
         const wrapper = shallow(
-            <Notification>Some test text</Notification>
+            <Notification icon={icon} content="content">Some test text</Notification>
```

Comment:
Иконка не обязательная

```suggestion
            <Notification content="content">Some test text</Notification>
```

Но теперь нужно проверить, что проставляются правильные иконки в зависимости от type (если конечно это видно в снепе).